bindings/azure/servicebusqueues/servicebusqueues_integration_test.go

```diff
@@ -60,7 +61,7 @@ func getMessageWithRetries(queue *servicebus.Queue, maxDuration time.Duration) (
 
 func TestQueueWithTTL(t *testing.T) {
```

Comment:
I wasn't able to get this to pass without changing the ttl. It seemed like the timing was too tight. Also, the docs don't recommend a ttl under 5 seconds. When I changed the ttl locally to 5 seconds I could get these to pass consistently.

Reply:
Good to know; it passed pretty reliably in the GitHub [runs](https://github.com/dapr/components-contrib/actions/runs/1247227218) and on my fork so I didn't see the flakiness, but if it's a timing issue, I'll bump it up to 5s in the #1173 to be safe.